clients/venice-push-job/src/main/java/com/linkedin/venice/hadoop/task/datawriter/AbstractPartitionWriter.java

```diff
@@ -91,6 +92,7 @@ public abstract class AbstractPartitionWriter extends AbstractDataWriterTask imp
   public static class VeniceWriterMessage {
     private final byte[] keyBytes;
     private final byte[] valueBytes;
+    private final long logicalts;
```

Comment:
Let's use consistent naming across all usages. I see we're using `logicalTimestamp` in some places and RMD in some. If we continue with `logicalts`, let's make it camel-case